### DISCUSSION TITLE
Automatic typing search in desktop client

### DISCUSSION THREAD

**Discussion by samschultzponsys** [ente-io/ente#4549] [Enhancements]
similar to "https://github.com/ente-io/ente/discussions/3264" when the ente client is open and is the window in focus it would be amazing if just typing would start filtering, then esc could clear the search at hand. This would help speed up lookup of codes massively. Thank you.

**Comment by kiler129** [comment]
Adding to this: a standard UX behavior of pressing escape to cancel/clear search would also be welcomed with this one.

**Comment by kiler129** [comment]
New version has a very similar option, auto-opening & focusing search on app start. It also auto-closes with "Escape" key :) While it's not *exactly* what @samschultzponsys asked, it solves the issue for me:

<img width="308" height="285" alt="image" src="https://github.com/user-attachments/assets/a6925842-36e5-4c54-a6e8-75471bd99744" />

**Comment by samschultzponsys** [comment]
I have just tested with that setting on and pressing escape does not zero out the search
